reset_password.php

```diff
@@ -0,0 +1,129 @@
+            $newconfirmpass=$_POST['confirm_password'];
+            $pass=password_hash($newpassword1,PASSWORD_BCRYPT);
+            if($newpassword1 == $newconfirmpass && $newpassword1 != '' && $newconfirmpass != '')
+            {
+                $updatequery ="UPDATE `users` SET `password` ='$pass' where `token` ='$tokennew' ";
+
+                $iquery =mysqli_query($conn, $updatequery);
+                if($iquery)
+                {
+                 
+                    $_SESSION['msg']= $lang['RECOVER_PASSWORD_MSG'];
+                    $updatequery1 ="UPDATE `users` SET `active` = '1' where `token` ='$tokennew' ";
+                    mysqli_query($conn, $updatequery1);
+                    header('location:login.php');
+                }
+                else
+                {
+                 $_SESSION['failedpass']= $lang['RECOVER_FAILED_MSG'];
+                  header('location: reset_password.php');
+                }
+            }
+
+            else if($newpassword1 != $newconfirmpass && $newpassword1 != '' && $newconfirmpass != '' )
+            {
+                $_SESSION['failedpass']= $lang['RECOVER_FAILED_MSG2'];
+                header('location: reset_password.php');    
+            }
+        }
+        else
+        {
+            echo $lang['TOKEN_NOT_FOUND'];
+        }      
+    }
+  }
+  else
+  {
+    $_SESSION['msg']="link expired try again connecting";
+    header('location:login.php');
+
+  }
+?>
+      <div style = "height:79vh">
+  <div class = "shadow-lg bg-white rounder" style= "margin:auto; margin-top:25px;margin-bottom: 40px;width: 40%; 
+  display: flex; height:300px; padding: 0;">
+                <div class="container mt-3">
+                <h4><?php echo $lang['RECOVER_HEADER']; ?></h4>
+                <form action="" method="post" id="form">   
+                 <p> 
+
+                   <?php
+                    if(isset($_SESSION['failedpass']))
+                    {
+                    echo $_SESSION['failedpass'];
+                    }
+                    else
+                    {
+                   echo  $_SESSION['failedpass']="";
+                    }
+                   ?>
+
+                 </p>
+                <div class="mb-3">
+                    <label for="exampleInputPassword1" class="form-label mt-2"><?php echo $lang['RECOVER_PWD_NEW']; ?></label>
+                    <input type="password" class="form-control" name ="pwd" id="pwd" required>
+                </div>
+                
+                <div class="mb-3">
+                    <label for="confirmpass" class="form-label"><?php echo $lang['RECOVER_PWD_REPEAT']; ?></label>
+                    <input type="password" class="form-control" name ="confirm_password" id="confirm_password" required>
+                </div>
+
+                 
+                <button type="submit"  name= "submit" class="btn btn-primary"><?php echo $lang['RECOVER_PWD_UPDATED']; ?></button>
```

Comment:
Please Change this variable @dheeth

Reply:
> Please Change this variable @dheeth

It's done @mahakbansal3 please check the next commit and confirm